src/taxonomy/TaxonomyListPage.jsx

```diff
@@ -64,10 +69,75 @@ const TaxonomyListHeaderButtons = () => {
   );
 };
 
+const OrganizationFilterSelector = ({
+  isOrganizationListLoaded,
+  organizationListData,
+  selectedOrgFilter,
+  setSelectedOrgFilter,
+}) => {
+  const intl = useIntl();
+  const isOrgSelected = (value) => (value === selectedOrgFilter ? <Check /> : null);
+  const selectOptions = [
+    <MenuItem
+      key="all-orgs-taxonomies"
+      className="x-small"
+      iconAfter={() => isOrgSelected('All taxonomies')}
```

Comment:
I think this should be a constant, like `const ALL_TAXONOMIES = "all_taxonomies"` 
Same with unassigned.

Reply:
Agreed, updated: https://github.com/openedx/frontend-app-course-authoring/pull/755/commits/737bfbd1484fad096c4b95da86150bb7d0249e12